packages/aws-rfdk/lib/deadline/lib/worker-configuration.ts

```diff
@@ -29,6 +29,26 @@ import {
   Version,
 } from './version';
 
+/**
+ * Provider for adding user data scripts
+ */
+export interface IInstanceUserDataProvider {
+  // Method that is invoked before configuration Cloud Watch Agent.
+  preCloudWatchAgent(host: IHost): void;
+  // Method that is invoked before Deadline worker and renderqueue configuration .
+  preDeadlineConfiguration(host: IHost): void;
```

Comment:
These can be made `readonly`

Reply:
'readonly' modifier can only appear on a property declaration or index signature.

Reply:
Ah my bad. Thanks for confirming.